src/components/PoiPopup.jsx

```diff
@@ -13,7 +53,17 @@ const PoiPopup = ({ poi }) => {
         fire('close_popup');
       }}
     >
-      <PoiItem poi={poi} withOpeningHours withImage inList />
+      <div className="u-mb-s">
+        <PoiItem poi={poi} withOpeningHours withImage inList />
+      </div>
+      <ActionButtons
```

Comment:
The button size should be reduced to match the design.
As we don't have the `small` size in our `Button` element, you probably have to overide some stuff in `popup.scss` like I did for text elements.